apps/charterafrica/src/payload/blocks/Explainers.js

```diff
@@ -0,0 +1,56 @@
+const Explainers = {
```

Comment:
Like the thinking here... but, do we need this?

I see explainers like posts (news, events, etc.). Are we just going to be pulling all explainers on the front-end?

This approach works when you need to limit/group collection items e.g. partners belonging to different groups.